lib/connection/pool.js

```diff
@@ -1035,6 +1038,16 @@ Pool.prototype.destroy = function(force) {
   checkStatus();
 };
 
+/**
+ * Reset all connections of this pool
+ *
+ * @param {function} [callback]
+ */
+Pool.prototype.reset = function(callback) {
+  // TODO: reset all connections
```

Comment:
Why is this still a TODO?

Reply:
Good call, this has been updated to include a `destroy` then `reconnect`, but should be more thoughtfully implemented in the final design